lookup_plugins/oo_option.py

```diff
@@ -17,7 +17,7 @@
 * if none of the above conditions are met, empty string is returned
 '''
 
-from ansible.utils import template
+from ansible import template
```

Comment:
This should probably be:
```
from ansible.template import template
```
The template module is now under the template package in Ansible 2.0+ and the call to template.template() below is getting swallowed by the broad try/except.

I believe the following may work for backwards compatibility as well:
```
from distutils.version import LooseVersion
if LooseVersion(ansible.__version__) > LooseVersion('2.0'):
    from ansible.template import template
else:
    from ansible.utils import template
```